src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -9,11 +10,18 @@
  (fn [{:keys [db]} token]
    {:db (assoc-in db [:wallet :ui :swap :asset-to-pay] token)
     :fx [[:dispatch
-          [:toasts/upsert
-           {:id   :swap-error
-            :type :negative
-            :text "Not implemented yet"}]]]}))
+          [:navigate-to :screen/wallet.swap-propasal]]
+         [:dispatch [:wallet.swap/init]]]}))
 
 (rf/reg-event-fx :wallet.swap/clean-asset-to-pay
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui :swap] dissoc :asset-to-pay)}))
+
+(rf/reg-event-fx :wallet.swap/init
```

Comment:
We might need to refactor event naming if we are going to have `:wallet.swap/start` and `:wallet.swap/init` events. Having both events with similar names looks confusing to me.

Reply:
Hi @briansztamfater, that's true. I have updated the new event name to `:wallet.swap/set-default-slippage`. When we start integrating with the backend, we can update the logic if the backend provides us with the default slippage value at that time. wdyt?